planetary-social/nos
# Fix issue loading authors on first launch

## Issues covered
#1024

## Description
Fixes the issue where authors weren't loaded when first launching the app with a new user account. Also right-aligns the small avatars of followers you know. Finally, fixed a layout issue when a NIP-05 was entered, but invalid, such as "example.com" which does not contain the `@` sign we expect.

## How to test
1. Logout of the app
2. Delete the app
3. Install the app
4. Create a new user account
5. Observe Discover tab

It should actually show profiles now. Previously, it was blank or mostly blank.

## Screenshots/Video

https://github.com/planetary-social/nos/assets/59564/d47fc12a-7d02-4df8-a4e7-d1c39b19f57f

And here's the fix for the invalid NIP-05 (which is "tedermusic.com"):

| Before | After |
|--------|--------|
| ![Simulator Screenshot - iPhone 15 Pro Max - 2024-04-30 at 15 37 44](https://github.com/planetary-social/nos/assets/59564/1fa6bfff-710c-4104-976d-e964eb352f2a) | ![Simulator Screenshot - iPhone 15 Pro Max - 2024-04-30 at 15 38 46](https://github.com/planetary-social/nos/assets/59564/f7862c69-8eb7-48b7-bb19-11ef0cce1fdf) |
